Dockerfile

```diff
@@ -44,5 +44,12 @@ RUN pnpm deploy --filter=paymaster-proxy --prod /prod/paymaster-proxy
 FROM base AS paymaster-proxy
 COPY --from=builder /prod/paymaster-proxy /prod/paymaster-proxy
 WORKDIR /prod/paymaster-proxy
+
+# setup certs for internal services
+RUN update-ca-certificates
```

Comment:
does this call need to go after `ENV NODE_EXTRA_CA_CERTS=/etc/ssl/certs/ca-certificates.crt`? I know that is the order we do it in gateway docker: https://github.com/ethereum-optimism/gateway/commit/af609004c08b99b09fedea359786d135b523e58c

Reply:
* `update-ca-certificates` will look under `/usr/local/share/ca-certificates` and concatenate the certs it finds into `/etc/ssl/certs/ca-certificates.crt`
* `NODE_EXTRA_CA_CERTS` tells node where to find any of our custom certs, since node ships with it's own set of certs it doesn't use `/etc/ssl/certs/ca-certificates.crt` by default. This value needs to be set before we execute the `pnpm start` command since that is when node will be looking for the env var